### DISCUSSION TITLE
ScriptReference/BuildPlayerOptions-locationPathName

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#135] [Notes]
# ScriptReference/BuildPlayerOptions-locationPathName

https://docs.unity3d.com/ScriptReference/BuildPlayerOptions-locationPathName.html

**Comment by vertxxyz** [comment]
This is the path to the executable, and is local to the project directory.  
When building for Windows an example path might look like:  
```csharp
// path = "Example"
locationPathName = Path.Combine("Builds", path, $"{Application.productName}.exe")
// Builds to "YourProject/Builds/Example" as "YourProductName.exe"
```
